### PR TITLE
Take 2: [WIP] CSV 1.8 and eventing 0.14

### PR DESCRIPTION
**Pull request by aliok**
1.8 CSV with eventing 0.14

* Do `make images dev`
* Go to OpenShift console, install serverless operator
* Install eventing (in knative-eventing namespace)

Image override for eventing 0.14 upgrade job work work. See
- Sandbox operator PR: https://github.com/knative-sandbox/operator/issues/76
- knative/eventing-operator PR: https://github.com/openshift-knative/eventing-operator/pull/27

### CODE REVIEW

File: olm-catalog/serverless-operator/1.8.0/serverless-operator.v1.8.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.13.2:knative-serving-webhook
+                    - name: IMAGE_3scale-kourier-gateway
+                      value: docker.io/maistra/proxyv2-ubi8:1.0.8
+                    - name: IMAGE_3scale-kourier-control
+                      value: quay.io/3scale/kourier:fix_duplication
+                    - name: IMAGE_eventing-controller_eventing-controller
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.14.0:knative-eventing-controller
+                    - name: IMAGE_eventing-webhook_eventing-webhook
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.14.0:knative-eventing-webhook
+                    - name: IMAGE_broker-controller_eventing_controller
```
Comment (aliok) [inline olm-catalog/serverless-operator/1.8.0/serverless-operator.v1.8.0.clusterserviceversion.yaml:362]:
```suggestion
                    - name: IMAGE_broker-controller_eventing-controller
```

**Comment by matzew** [comment]
@aliok 

```
        knative_eventing_test.go:48: Deployment count in namespace knative-eventing is not the same as expected 5: Deployment count in namespace knative-eventing did not reach the expected count 5 in time: timed out waiting for the condition
```

we have more, now ;-)

**Comment by markusthoemmes** [comment]
Shall we split out the 1.8.0 CSV maybe? We can keep it pointing to 0.13 at first and then ship 0.14 artifacts in a separate PR. Unblocks a few things we wanted to do once the 1.8.0 CSV lands.

**Comment by matzew** [comment]
> Shall we split out the 1.8.0 CSV maybe? We can keep it pointing to 0.13 at first and then ship 0.14 artifacts in a separate PR. Unblocks a few things we wanted to do once the 1.8.0 CSV lands.

I was wondering the same, a bit early.

If we just do:
* bump to 1.8.0 with 0.13
* later update (eventing) 0.14 

/cc @aliok

**Comment by aliok** [comment]
sounds good. @matzew is doing it now

**Comment by aliok** [comment]
/close

Closing this now. Once https://github.com/openshift-knative/serverless-operator/pull/295 and https://github.com/openshift-knative/serverless-operator/pull/296 is merged, we can create a new PR that modifies 1.8 CSV with eventing 0.14

**Pull request closed without merging**